src/main/resources/db/migration/V23__create_email_template_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE email_templates (
+ id VARCHAR(36) PRIMARY KEY,
+ title VARCHAR(255) NOT NULL UNIQUE,
+ template TEXT NOT NULL,
+ status VARCHAR(255),
```

Comment:
there is a V23 of the migration file

Reply:
I didn't touch any other version of the migration file.
I just did my own which is v25.